Constant UINT declaration causing GO Compiler to fail on x86

__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. make project on any 32 bit system
2. Compiler will complain with the following

`cmd/influx/setup.go:319:54: constant 3600000000000 overflows uint`
`make: *** [Makefile:82: bin/linux/influx] Error 2`

Line 319 in setup.go is the following, Appears that time.Hour is a 64 bit constant.
`rp = fmt.Sprintf("%d hrs", or.RetentionPeriod/uint(time.Hour))`

__Expected behavior:__
Should handle 32 bit architectures

__Actual behavior:__
Fails to compile due to overflow on uint

__Environment info:__

Ubuntu 19.10 i386/x86
Node 14
Yarn 1.22
Go 1.13.12.linux-386

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -13,6 +13,7 @@
 1. [18595](https://github.com/influxdata/influxdb/pull/18595): Add ability to skip resources in a template by kind or by metadata.name
 1. [18600](https://github.com/influxdata/influxdb/pull/18600): Extend influx apply with resource filter capabilities
 1. [18601](https://github.com/influxdata/influxdb/pull/18601): Provide active config  running influx config without args
+1. [18602](https://github.com/influxdata/influxdb/pull/18602): Fix uint overflow during setup on 32bit systems
 
 ## v2.0.0-beta.12 [2020-06-12]
 

--- a/cmd/influx/setup.go
+++ b/cmd/influx/setup.go
@@ -316,7 +316,7 @@ func getConfirm(ui *input.UI, or *influxdb.OnboardingRequest) bool {
 	for {
 		rp := "infinite"
 		if or.RetentionPeriod > 0 {
-			rp = fmt.Sprintf("%d hrs", or.RetentionPeriod/uint(time.Hour))
+			rp = fmt.Sprintf("%d hrs", time.Duration(or.RetentionPeriod)/time.Hour)
 		}
 		ui.Writer.Write(promptWithColor(fmt.Sprintf(`
 You have entered:

